app/models/manageiq/providers/ansible_tower/configuration_manager/refresh_worker/collector.rb

```diff
@@ -0,0 +1,19 @@
+class ManageIQ::Providers::AnsibleTower::ConfigurationManager::RefreshWorker::Collector
+  def initialize(ems, target, _options = nil)
+    @ems        = ems
+    @connection = ems.connect
+    @target     = target
```

Comment:
also, I am getting into a point that targeted refresh goes through managers

so e.g. a proper targeted refresh of a VM needs info from CloudManager, NetworkManager and StorageManager now